Documentation/general/cli-ux-guidelines.md

```diff
@@ -0,0 +1,155 @@
+.NET Core Command-Line Tools UX Guidelines
+-------------------------------------------
+
+This document will cover the CLI UX guidelines that anyone adding or suggesting

```

Comment:
This could be simpler, like:

> This document defines user experience guidelines for .NET Core CLI commands. It is intended for anyone creating or updating .NET Core CLI commands. It is recommended reading for anyone considering a pull request to this repository.